plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlTypeMapping.java

```diff
@@ -621,13 +621,13 @@ public void testUnsupportedTime(String unsupportedTime)
         try (TestTable table = new TestTable(memSqlServer::execute, "tpch.test_unsupported_time", "(col time)", ImmutableList.of(format("'%s'", unsupportedTime)))) {
             assertQueryFails(
                     "SELECT * FROM " + table.getName(),
-                    format("\\Q%s cannot be parse as LocalTime (format is \"HH:mm:ss[.S]\" for data type \"TIME\")", unsupportedTime));
+                    format("Text '%s' could not be parsed.*", unsupportedTime));
```

Comment:
As per comment in `MemSqlTimeColumnMapping`, I'll revert this change to keep the new message as close to  the original as possible (I'll fix "parse"->"parsed" typo).

Reply:
I consider this a regression since the new message is not as useful for end-user. It just tells could not be parsed but not why or how to fix it.

Reply:
The underlying reason is part of the `%s`, so I wouldn't consider it a regression - but I agree we can keep the original message almost unchanged.
👍